### PR TITLE
fix(Coinify): style fiat input dropdown

### PR DESCRIPTION
**Pull request by prwelber**
## Description
Add a concise explanation of the changes.

## Change Type
Please enter one or more of the following: 
- Feature
- Bug Fix
- CI/CD
- Upgrades
- Other (add explanation)

## Testing Steps
Detail the steps required for the reviewer(s) to verify and test these changes.

## Code Checklist
- [x] Code compiles successfully (verified via `yarn start`)
- [x] No lint issues exist (verified via `yarn lint`)
- [x] New and existing unit tests pass (verified via `yarn test`)
- [x] `README.md` and other documentation is updated as needed

### CODE REVIEW

File: packages/blockchain-info-components/src/Form/SelectInput/template.js [hunk 1]
```diff
@@ -12,11 +12,28 @@ const StyledSelect = styled(Select)`
   .control {
     box-shadow: none;
     border: 1px solid ${props => props.theme[props.borderColor]};
+    border-left: ${props => props.asideTextInput && 'none'};
```
Comment (prwelber) [inline packages/blockchain-info-components/src/Form/SelectInput/template.js]: @plondon should these styles be applied elsewhere? feels weird to have all this very situation specific css in the main components repo.

Comment (tony-blockchain) [reply packages/blockchain-info-components/src/Form/SelectInput/template.js]: absolutely, you can wrap `SelectBox`/`SelectInput` with `styled` and use `.control` selector

Comment (tony-blockchain) [reply packages/blockchain-info-components/src/Form/SelectInput/template.js]: asideTextInput should be removed in this case

Comment (plondon) [reply packages/blockchain-info-components/src/Form/SelectInput/template.js]: You can make your own dropdown by extending this one. This should not be here.

File: packages/blockchain-wallet-v4-frontend/src/components/Form/SelectBoxCoinifyCurrency/index.js [hunk 2]
```diff
@@ -12,6 +12,32 @@ const CustomSelectBox = styled(SelectBox)`
   * {
     border-left: none;
   }
+  .control {
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/components/Form/SelectBoxCoinifyCurrency/index.js]: I think this should be bc__control now

Comment (prwelber) [reply packages/blockchain-wallet-v4-frontend/src/components/Form/SelectBoxCoinifyCurrency/index.js]: Thanks, fixed this.

File: packages/blockchain-wallet-v4-frontend/src/scenes/BuySell/CoinifyCheckout/Content/QuoteInputTemplate.js [hunk 3]
```diff
@@ -326,6 +326,7 @@ const FiatConvertor = props => {
             component={SelectBoxCoinifyCurrency}
             defaultDisplay={defaultCurrency}
             isSell={isSell}
+            asideTextInput
```
Comment (tony-blockchain) [inline packages/blockchain-wallet-v4-frontend/src/scenes/BuySell/CoinifyCheckout/Content/QuoteInputTemplate.js]: this prop looks like it's unneeded

File: packages/blockchain-wallet-v4-frontend/src/components/Form/SelectBoxCoinifyCurrency/index.js [hunk 4]
```diff
@@ -12,6 +12,32 @@ const CustomSelectBox = styled(SelectBox)`
...
+      border-left: none;
+    }
+    &:active {
+      border: 1px solid ${props => props.theme['gray-2']};
+      border-left: none;
+    }
+  }
 `
 
 class SelectBoxCoinifyCurrency extends React.PureComponent {
```
Comment (1Bennell) [inline packages/blockchain-wallet-v4-frontend/src/components/Form/SelectBoxCoinifyCurrency/index.js]: Class

File: packages/blockchain-wallet-v4-frontend/src/components/Form/SelectBoxCoinifyCurrency/index.js [hunk 5]
```diff
@@ -12,6 +12,28 @@ const CustomSelectBox = styled(SelectBox)`
...
+    }
+    &:hover {
+      border: 1px solid ${props => props.theme['gray-2']};
+      border-left: none;
+    }
+    &:active {
+      border: 1px solid ${props => props.theme['gray-2']};
+      border-left: none;
+    }
+  }
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/components/Form/SelectBoxCoinifyCurrency/index.js]: Please also target bc__single-value and make sure the font-size and color is the same as the other field's coin type

Comment (prwelber) [reply packages/blockchain-wallet-v4-frontend/src/components/Form/SelectBoxCoinifyCurrency/index.js]: Woops, sorry about that. Should be fixed now for the selected currency and items in the dropdown.

**Pull request merged**